### PR TITLE
Multi domain manifest

### PR DESCRIPTION
**Pull request by andreban**

### CODE REVIEW

File: demos/twa-multi-domain/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <meta-data
+            android:name="asset_statements"
+            android:resource="@string/asset_statements" />
+
+        <activity android:name="com.google.androidbrowserhelper.trusted.LauncherActivity"
+            android:label="@string/app_name">
+
+            <!-- Additional origins to be validated with Digital AssetLinks are listed under
+                this meta-tag, which points to a string-array in strings.xml -->
+            <meta-data android:name="android.support.customtabs.trusted.DOMAINS_TO_VALIDATE"
```
Comment (PEConn) [inline demos/twa-multi-domain/src/main/AndroidManifest.xml]: This needs to be updated right (the DOMAINS_TO_VALIDATE)?

File: demos/twa-multi-domain/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                android:value="@array/additional_trusted_origins" />
+
+            <meta-data android:name="android.support.customtabs.trusted.ADDITIONAL_TRUSTED_ORIGINS"
```
Comment (PEConn) [inline demos/twa-multi-domain/src/main/AndroidManifest.xml]: Now I'm confused with what you're doing... Aren't you meant to have ADDITIONAL_TRUSTED_ORIGINS set to @array/additional_trusted_origins ?

File: demos/twa-multi-domain/build.gradle [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    api 'androidx.browser:browser:1.2.0-alpha07'
+    implementation project(path: ':androidbrowserhelper')
+    implementation 'androidx.appcompat:appcompat:1.0.2'
+
+    testImplementation 'junit:junit:4.12'
+    androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
```
Comment (PEConn) [inline demos/twa-multi-domain/build.gradle]: You don't need the lines about testing dependencies.

File: demos/twa-multi-domain/build.gradle [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    api 'androidx.browser:browser:1.2.0-alpha07'
+    implementation project(path: ':androidbrowserhelper')
+    implementation 'androidx.appcompat:appcompat:1.0.2'
```
Comment (PEConn) [inline demos/twa-multi-domain/build.gradle]: I don't think you need this dependency.

File: demos/twa-multi-domain/build.gradle [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    api 'androidx.browser:browser:1.2.0-alpha07'
```
Comment (PEConn) [inline demos/twa-multi-domain/build.gradle]: I don't think you need this dependency either.

File: demos/twa-multi-domain/build.gradle [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
```
Comment (PEConn) [inline demos/twa-multi-domain/build.gradle]: You could probably get rid of these lines and proguard-rules.pro if we want to keep things simple.

File: demos/twa-multi-domain/build.gradle [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+
+    defaultConfig {
+        applicationId "com.google.browser.examples.twa_multi_domain"
+        minSdkVersion 26
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```
Comment (PEConn) [inline demos/twa-multi-domain/build.gradle]: We have no tests! Can get rid of this.

File: demos/twa-multi-domain/src/main/AndroidManifest.xml (hunk 2 above)
Comment (andreban) [reply demos/twa-multi-domain/src/main/AndroidManifest.xml]: Yep - I changed the wrong key when renaming. This one was supposed to be LAUNCH_URL

File: demos/twa-multi-domain/src/main/AndroidManifest.xml (hunk 1 above)
Comment (andreban) [reply demos/twa-multi-domain/src/main/AndroidManifest.xml]: Yes.

File: demos/twa-multi-domain/build.gradle (hunk 7 above)
Comment (andreban) [reply demos/twa-multi-domain/build.gradle]: done

File: demos/twa-multi-domain/build.gradle (hunk 6 above)
Comment (andreban) [reply demos/twa-multi-domain/build.gradle]: done.

File: demos/twa-multi-domain/build.gradle (hunk 5 above)
Comment (andreban) [reply demos/twa-multi-domain/build.gradle]: done.

File: demos/twa-multi-domain/build.gradle (hunk 4 above)
Comment (andreban) [reply demos/twa-multi-domain/build.gradle]: Kind of - It does build and run, but Android Studio marks some things as invalid in the UI. I'm, removing, as I think this is more like an Android Studio issue and shouldn't be here anyway.

File: demos/twa-multi-domain/build.gradle (hunk 3 above)
Comment (andreban) [reply demos/twa-multi-domain/build.gradle]: done

**Pull request merged**